index.html

```diff
@@ -320,9 +336,62 @@ <h2>Skills</h2>
                     </article>
                 </article>
             </article>
+            <!-- contact section -->
+            <section class="main-container bottom" >
+                <section class="bottomcontainer">
+                    <div class="contact-upper-bar">
+                        <p class="contact-heading">
+                            I'm always interested in hearing 
+                            about new projects, so if you'd 
+                            like to chat please get in touch.
+                        </p>
+                    </div>
+                    <div class="contact-bottom-bar">
+                        <form action="https://formspree.io/f/mlekqgrr" method="POST">
+                            <div class="input-box">
+                                <input type="text" class="input" name="full-name" placeholder="Full name" maxlength="30s">
```

Comment:
- [ ] Please fix the type in your `maxlength` attribute, it should be **30** not **30s** in order to restrict the input to take more than 30 characters full name of the user.

Reply:
Hello, @skyv26 Thank You for your comments. Concerning placing the images as background I have tried it and they are not displaying as expected. Could I be doing it wrongly?

Reply:
@Bettenoch Yes, if you are not getting the desired results, it means that you are doing something wrong or not properly. Please do focus and try to understand, I believe you can do it too ❤️

Reply:
> @Bettenoch Yes, if you are not getting the desired results, it means that you are doing something wrong or not properly. Please do focus and try to understand, I believe you can do it too ❤️

Thank you @skyv26 for recommending the changes I have used the images as background images and indeed they are working efficiently.

Reply:
Wonderful 😊 @Bettenoch